odoo/addons/base/models/res_company.py

```diff
@@ -119,6 +120,10 @@ def _get_company_address_update(self, partner):
         return dict((fname, partner[fname])
                     for fname in self._get_company_address_field_names())
     
+    def _get_fiscalyear_last_day(self, year):
+        """ Return a last day of the February month based on year."""
+        return calendar.monthrange(year, 2)[1]
```

Comment:
This is related to accounting only, please keep everything in `account` or modules depending on it

Reply:
How does the method name make sense with it's return value and it's argument?

Reply:
Hello,

Do you mean the function name should be like `_get_february_last_day`?

Reply:
I'm saying that the method should do what the naming implies:
> read `fiscalyear_last_day` and `fiscalyear_last_month` and give that date for the `year` provided, while managing leap years.

You can find a better name if you want to.